integration-auth/src/main/java/org/symphonyoss/integration/authentication/AuthenticationProxyImpl.java

```diff
@@ -176,42 +180,43 @@ public String getSessionToken(String configurationId) {
    * same exception
    * @param userId
    * @param code
-   * @param e
+   * @param exception
    * @throws RemoteApiException the original exception
    */
   @Override
-  public synchronized void reAuthOrThrow(String userId, int code, Exception e)
+  public synchronized void reAuthOrThrow(String userId, int code, Exception exception)
```

Comment:
We might change this exception to RemoteApiException and remove the 'code' parameter.